recipes/libcheck/all/conanfile.py

```diff
@@ -48,14 +53,14 @@ def requirements(self):
             self.requires("subunit/1.4.0")
 
     def build_requirements(self):
-        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
-            self.build_requires("msys2/20200517")
+        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
         if self.settings.compiler == "Visual Studio":
-            self.build_requires("automake/1.16.2")
+            self.build_requires("automake/1.16.3")
```

Comment:
Why does this recipe need msys2 and automake, while build is based on CMake?

Why pkg_config generator while pkgconf is not in build requirements?

Reply:
This was probably a rest over from autotools usage.
Removed it.